MassEffectModManagerCore/modmanager/localizations/fra.xaml

```diff
@@ -274,375 +274,376 @@
-    <system:String x:Key="string_importMods">Import mods</system:String>
-    <system:String x:Key="string_compressPackages">Compress packages</system:String>
-    <system:String x:Key="string_unselectAll">Unselect all</system:String>
-    <system:String x:Key="string_selectAll">Select all</system:String>
-    <system:String x:Key="string_selectAModOnTheLeftToViewItsDescription">Select a mod on the left to view its description</system:String>
-    <system:String x:Key="string_selectModsToImportIntoModManagerLibrary">Select mods to import into Mod Manager library</system:String>
-    <system:String x:Key="string_noCompatibleModsFoundInArchive">No compatible mods found in archive</system:String>
-    <system:String x:Key="string_executableModsMustBeValidatedByME3Tweaks" xml:space="preserve">Executable installer based mods must be validated by ME3Tweaks before they can be imported into Mod Manager This is to prevent breaking third party mods.\n\nThis executable has not been validated. Check for a ME3Tweaks Mod Manager compatible version from the download page, or ask the developer to make a ME3Tweaks Mod Manager compatible version.</system:String>
-    <system:String x:Key="string_noCompatibleModsFoundInArchiveExtended">No compatible mods found in archive. If this is a texture mod, you must install it with ALOT Installer. If this is a known Mod Manager mod, please come to the ME3Tweaks Discord from the Help menu.</system:String>
-    <system:String x:Key="string_interp_scanningX">Scanning {0}</system:String>
+    <system:String x:Key="string_importModsFromArchiveFile">Importer des mods à partir d'un fichier d'archive</system:String>
+    <system:String x:Key="string_importMods">Importer les mods</system:String>
+    <system:String x:Key="string_compressPackages">Compresser les paquets</system:String>
+    <system:String x:Key="string_unselectAll">Désélectionner tout</system:String>
+    <system:String x:Key="string_selectAll">Sélectionner tout</system:String>
+    <system:String x:Key="string_selectAModOnTheLeftToViewItsDescription">Sélectionnez un mod à gauche pour voir sa description</system:String>
+    <system:String x:Key="string_selectModsToImportIntoModManagerLibrary">Sélectionnez les mods à importer dans la bibliothèque Mod Manager</system:String>
+    <system:String x:Key="string_noCompatibleModsFoundInArchive">Pas de mods compatibles trouvés dans l'archive</system:String>
+    <system:String x:Key="string_executableModsMustBeValidatedByME3Tweaks" xml:space="preserve">Les mods basés sur un installateur exécutable doivent être validés par ME3Tweaks avant de pouvoir être importés dans Mod Manager, afin d'éviter de casser des mods tiers.\n\nCet exécutable n'a pas été validé. Vérifiez si une version compatible avec ME3Tweaks Mod Manager est disponible sur la page de téléchargement, ou demandez au développeur de faire une version compatible avec ME3Tweaks Mod Manager.</system:String>
```

Comment:
The INT line for this is supposed to be "Executable installer based mods must be validated by ME3Tweaks before they can be imported into Mod Manager.", a period on the end. I just fixed it on my end (it was previously missing), so the translation might be slightly off.